lib/sftp.js

```diff
@@ -3,7 +3,34 @@ const {def, ConnectionConfig, ReadStreamType, SshClientType} = require("./sanctu
 const {FutureType} = require("fluture-sanctuary-types");
 const {Future} = require("fluture");
 
-// sendFileViaSftp :: String -> ConnectionConfig -> Future
+const getFileViaSftp =
+  def("getFileViaSftp")
+  ({})
+  ([SshClientType, ConnectionConfig, FutureType($.String)($.Any)])
+  (client => connectionConfig => Future((reject, resolve) => {
+    client.on("error", (err) => {
+      client.end();
+      reject(err);
+    });
+
+    client.on("ready", () => {
+      client.sftp((err, sftp) => {
+        if (err) {
+          client.end();
+          return reject(err);
+        }
+
+        try {
+          resolve(sftp.createReadStream(connectionConfig.remoteFilePath));
```

Comment:
Does this work okay?  This seems safer for some reason, but I am honestly not sure if it matters.
```suggestion
          const sftpStream = sftp.createReadStream(connectionConfig.remoteFilePath);
          resolve(sftpStream);
```

Reply:
I'm going to try `encase`. Had a question out on the fluture gitter page and I think the response will work here

Reply:
Looks like encase will work just fine here.  When we integrate orpheus, we will need to change this though and return our `logObj` on reject.

Reply:
encase and attempt both caused major code hangage so I went with your suggestion here